client/components/webpack-build-monitor/index.jsx

```diff
@@ -111,7 +111,7 @@ class WebpackBuildMonitor extends React.Component {
 	componentDidMount() {
 		this.unsubscribe = store.subscribe( () => {
 			const status = store.getState();
-			if ( status !== this.state.status ) {
+			if ( ! isEqual( status, this.state ) ) {
```

Comment:
`PureComponent` only relates to props right?  Also i'm not sure if `combineReducers` returns the same object every action does it (in which case an identity check would never be true)